builtin/providers/aws/resource_aws_ecr_repository.go

```diff
@@ -81,9 +87,17 @@ func resourceAwsEcrRepositoryRead(d *schema.ResourceData, meta interface{}) erro
 	d.Set("arn", *repository.RepositoryArn)
 	d.Set("registry_id", *repository.RegistryId)
 
+	repositoryUrl := buildRepositoryUrl(repository, meta.(*AWSClient).region)
+	log.Printf("[INFO] Setting the repository url to be %s", repositoryUrl)
+	d.Set("repository_url", repositoryUrl)
+
 	return nil
 }
 
+func buildRepositoryUrl(repo *ecr.Repository, region string) string {
+	return fmt.Sprintf("https://%s.dkr.ecr.%s.amazonaws.com/%s", *repo.RegistryId, region, *repo.RepositoryName)
```

Comment:
I have the same issue. I'd like to use this in a container definition template and the HTTPS breaks it.